apps/src/sites/code.org/pages/public/student/middle-high.js

```diff
@@ -1,23 +1,9 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import {Provider} from 'react-redux';
-
-import MiddleHighResourceCards from './MiddleHighResourceCards';
 import initResponsive from '@cdo/apps/code-studio/responsive';
-import isRtl from '@cdo/apps/code-studio/isRtlRedux';
-import responsive from '@cdo/apps/code-studio/responsiveRedux';
-import {getStore, registerReducers} from '@cdo/apps/redux';
-
-registerReducers({isRtl, responsive});
 
 document.addEventListener('DOMContentLoaded', () => {
   initResponsive();
 
-  const container = document.getElementById('educate-resources-grid');
-  ReactDOM.render(
-    <Provider store={getStore()}>
-      <MiddleHighResourceCards />
-    </Provider>,
-    container
-  );
+  $(document).ready(function() {
+    $('.hoc-tiles-container').load('/dashboardapi/middle_high_student_labs');
```

Comment:
do we have a specific motivation for loading this content asynchronously, rather than including it directly in the template as a partial?

Reply:
Oh, this is because we're rendering in pegasus but pulling content served by dashboard.